test/infrastructure/docker/docker/machine.go

```diff
@@ -104,6 +104,15 @@ func (m *Machine) ProviderID() string {
 	return fmt.Sprintf("docker:////%s", m.ContainerName())
 }
 
+func (m *Machine) Address() (string, error) {
+	ipv4, _, err := m.container.IP(context.TODO())
```

Comment:
We do have a context in the calling function that you could pass through - want to do that instead of using TODO here?

Reply:
yeah good catch

Reply:
updated